### PR TITLE
Create so-you're-going-on-vacation.md

### PR DESCRIPTION
**Pull request by joshblack**
Adds a checklist for folks before they go on vacation.

#### Changelog

**New**

- `so-you're-going-on-vacation.md` with a checklist before going on vacation 🎉 

**Changed**

**Removed**

### CODE REVIEW

**Review by alisonjoseph — APPROVED** [review]
🎉

**Comment by dakahn** [comment]
😆 You actually used my file name!

**Comment by abbeyhrt** [comment]
Should we add something like "...and make sure to remove yourself from the Excluded team members when you get back" to the part about pull panda or nah? Might be an assumed thing!

File: docs/guides/so-you're-going-on-vacation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+You can do this in your email client by heading to the top-right hand side and
+selecting your profile image, then going to **Mail and calendar settings**.
+
+In this window, you can go to **Out of Office** and set the relevant dates and
+message you would like to have sent out when folks email you.
+
+- [ ] Set your Slack status
+
+Slack has a setting for setting your status for a pre-determined about of time,
```
Comment (jnm2377) [inline docs/guides/so-you're-going-on-vacation.md]: should this say:
> pre-determined _amount_ of time

**Review by jnm2377 — APPROVED** [review]

File: docs/guides/so-you're-going-on-vacation.md (hunk 1 above)
Comment (joshblack) [reply docs/guides/so-you're-going-on-vacation.md]: 👍 want to use the suggestion syntax and I can commit with your changes? 👀

**Comment by joshblack** [comment]
@abbeyhrt I'd be down, want to use the suggestion thing and I can commit it?

File: docs/guides/so-you're-going-on-vacation.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+what you choose as your status is up to you! Some common ones include the
+defaults from slack, or `OOO until XX-YY-ZZZZ`.
+
+- [ ] Exclude yourself as a reviewer for your team on Pull Panda
+
+To make sure reviews aren't being requested while you're on vacation, make sure
+to head on over to [Pull Panda](https://pullpanda.com) and go to the
+**Assigner** tab. From there, you can select your team. For each team there is
+an **Excluded team members** section where you can add your name while you're
+out!
```
Comment (abbeyhrt) [inline docs/guides/so-you're-going-on-vacation.md]:
```suggestion
out. Don't forget to remove yourself from the Excluded team members section  when you get back!
```

**Review by abbeyhrt — APPROVED** [review]

**Comment by alisonjoseph** [comment]
Do we want to add links to any of these things? Or no because its all internal links?

**Review by jeanservaas — APPROVED** [review]

**Review by janchild — APPROVED** [review]
Looks great! Nice additions, @abbeyhrt !

**Comment by joshblack** [comment]
@alisonjoseph I added one for vPlanner, but for the other ones there actually aren't any links surprisingly 🤔 or at least none that I know of. If you find them let me know and I can do an ibm.biz url

**Pull request merged**